src/AgentImplementation.sol

```diff
@@ -125,7 +119,7 @@ contract AgentImplementation is IAgent, ERC721Holder, ERC1155Holder {
         }
     }
 
-    function _executeLogics(IParam.Logic[] calldata logics, bool shouldChargeFeeByLogic) internal {
+    function _executeLogics(IParam.Logic[] calldata logics, bool chargeFeeOnCallback) internal {
```

Comment:
I think I need this parameter to decide whether to charge fee while doing transfer also, but it's fine to name it in this way first.